include/class.format.php

```diff
@@ -514,13 +514,21 @@ function __formatDate($timestamp, $format, $fromDb, $dayType, $timeType,
 
         if (!$timestamp)
             return '';
+	
+	// We need this codepiece to make sure we have our timestamp in 'Y-m-d H:i:s' format, as sometimes it is given to this function as unixtime
+	//TODO fix that!
+	if (is_numeric($timestamp)) {
+		$tmp = new DateTime();
+		$timestamp = $tmp->setTimestamp($timestamp)->format('Y-m-d H:i:s');
```

Comment:
fdellwing  - This is very helpful. In my case, system_time_zone in MySQL picks up "AST" from the host, and with your fix, the time is correct on our tickets, but bottom of the page in /scp/system.php now says "Timezone: America/Santo_Domingo (Interpreted as Asia/Riyadh)".